bridge/slack/helpers_test.go

```diff
@@ -0,0 +1,28 @@
+package bslack
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExtractTopicOrPurpose(t *testing.T) {
+	testcases := []struct {
```

Comment:
We should actually make this a `map[string]*[]struct` so that we can name test cases.

Reply:
That's a great idea :) after all, won't always fit in the text portion